lib/broadside/command.rb

```diff
@@ -34,22 +34,6 @@ def targets(options)
         info "\n" + table.render(:ascii, padding: [0, 1])
       end
 
-      def deploy_short(options)
```

Comment:
i was thinking that ultimately these methods would live on a `Deploy` module, not a `Command` module that just passes them through... so they didn't really belong here.  thoughts?